react/components/Table/index.js

```diff
@@ -97,9 +97,18 @@ class Table extends PureComponent {
     )
   }
 
+  handleSelectionChange = () => {
+    if (this.props.bulkActions && this.props.bulkActions.onChange) {
+      const selectedParameters = this.state.allLinesSelected
+        ? { allLinesSelected: true }
+        : { selectedRows: this.state.selectedRows }
+        this.props.bulkActions.onChange(selectedParameters)
+    }
+  }
+
   handleSelectAllLines = () => {
     const { items: selectedRows } = this.props
-    this.setState({ selectedRows, allLinesSelected: true })
+    this.setState({ selectedRows, allLinesSelected: true }, this.handleSelectionChange)
```

Comment:
Is it the same as `this.setState({ selectedRows, allLinesSelected: true }, () => this.handleSelectionChange)`?

Reply:
Yes, but without using an anonymous function